lib/omniauth/strategies/drip.rb

```diff
@@ -17,7 +17,7 @@ class Drip < OmniAuth::Strategies::OAuth2
         :token_url => 'https://www.getdrip.com/oauth/token'
       }
 
-      uid { access_token.client.id }
+      uid { raw_info['accounts'][0]['id'] }
```

Comment:
Yeah, I see what you mean. I need to give this some thought. The complication is that most tokens issued these days are indeed always tied to one (and only one) active account, but there are some edge cases.

In the meantime, I would recommend making a call to the `/accounts` endpoint right after obtaining the token to figure out which account the token is authorized to access. This is what most integrators do, which is probably why this issue has not be raised to date.